ci/jupyter.js

```diff
@@ -6,12 +6,41 @@ const { sleep } = require("./sleep");
 global.XMLHttpRequest = require("xmlhttprequest").XMLHttpRequest;
 const { ajax } = require("rxjs/ajax");
 
+let url_path_join = function(...pieces) {
```

Comment:
It would be good to move this function to a utility file as per our discussion.

Reply:
Actually, may we should consider importing [url-join](https://www.npmjs.com/package/url-join).